packages/common/src/ui/components/modals/InputModal/InputModal.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import {
+  BasicSpinner,
+  Box,
+  useDialog,
+  DialogButton,
+} from '@openmsupply-client/common';
+
+interface InputModalProps {
+  isOpen: boolean;
+  Input: React.ReactNode;
+  onClose: () => void;
+  title: string;
+  onChange: (() => Promise<void>) | (() => void);
+}
+
+export const InputModal = ({
```

Comment:
<img width="844" alt="image" src="https://user-images.githubusercontent.com/35858975/158007646-ea4f7c0d-a254-4eb3-b885-a3a2d6f5cca0.png">

1. Not the best looking. Though.. looks OK.
2. Bad naming? https://mui.com/components/dialogs/#form-dialogs going for this type of thing
3. Bad location? `Inputs` better?

Reply:
1. Looks ok to me
2. Name is fine - good to have 'modal' instead of dialog / modal
3. Could be in either! Modal is better because this is more than just an input
😄